### DISCUSSION TITLE
Custom PATH environment Variable Per Profile

### DISCUSSION THREAD

**Discussion by 136205** [Eugeny/tabby#2987] [General]
How do I specify a custom path value which includes the current path plus some additional directories to add to the path, per profile?

For reasons, we are using different versions of Gradle. In one profile, I want to set the PATH to %PATH%;c:\gradle.v1\bin. And in another profile, I want to the set path to %PATH%;c:\gradle.v2\bin. Terminus doesn't seem to be expanding the value of %PATH%. Instead I get the literal value of '%PATH%;c:\gradle.v1\bin'

Is this supported in Terminus?

**Comment by Eugeny** [comment]
Currently not supported - added it as a feature request, thanks: #2994
